### ISSUE TITLE
Ordered lists do not render correctly in `solution` box

### ISSUE THREAD

**Issue by tobyhodges** [carpentries/sandpaper#323]
```Markdown
::: challenge

1. Monday
1. Tuesday
1. Wednesday
1. Thursday
1. Friday

:::::: solution

1. Saturday
1. Sunday

::::::

:::
```

becomes in the built page:

![Screenshot of a challenge and solution block on an episode page, with the days of the week Monday through Friday appearing as a numbered list in the challenge body, and Saturday and Sunday listed without item markers in the solution body.](https://user-images.githubusercontent.com/9694524/179211582-cfdde650-910c-4837-888c-a323ae86b8c2.png)

**Comment by tobyhodges** [comment]
The `<ol><li>` elements seem to be created correctly in the HTML, so maybe this issue belongs in https://github.com/carpentries/varnish/?

**Comment by zkamvar** [comment]
Yup, this is definitely an issue for {varnish}. I'll transfer the issue.
